test/powershell/Language/Parser/AutomaticVariables.Tests.ps1

```diff
@@ -2,18 +2,6 @@
 # Licensed under the MIT License.
 
 Describe 'Automatic variable $input' -Tags "CI" {
-    # Skip on hold for discussion on https://github.com/PowerShell/PowerShell/issues/1563
-    # $input type in advanced functions
-    It '$input Type should be enumerator' -Skip {
-        function from_begin { [cmdletbinding()]param() begin { Write-Output -NoEnumerate $input } }
-        function from_process { [cmdletbinding()]param() process { Write-Output -NoEnumerate $input } }
-        function from_end { [cmdletbinding()]param() end { Write-Output -NoEnumerate $input } }
-
-        (from_begin) -is [System.Collections.IEnumerator] | Should -BeTrue
```

Comment:
Based on the issue and committee decision, it seems that instead of removing this test, we should just change lines 12-14 to reflect the current design?

```powershell
(from_begin) -is [System.Collections.ArrayList] | Should Be $true
(from_process) -is [System.Collections.ArrayList] | Should Be $true
(from_end) -is [System.Object[]] | Should Be $true
```